--- itop-2.5.1/web/application/itopwebpage.class.inc.php	2018-10-17 13:07:36.000000000 +0800
+++ web/application/itopwebpage.class.inc.php	2018-12-03 16:41:14.170766516 +0800
@@ -192,7 +192,11 @@ EOF;
         $sTimeFormat = AttributeDateTime::GetFormat()->ToTimeFormat();
         $oTimeFormat = new DateTimeFormat($sTimeFormat);
         $sJSLangShort = json_encode(strtolower(substr(Dict::GetUserLanguage(), 0, 2)));
-
+        // 中文bug
+        if($sJSLangShort == '"zh"')
+        {
+            $sJSLangShort = '"zh-CN"';
+        }
         // Date picker options
         $aPickerOptions = array(
             'showOn' => 'button',
