--- itop-2.5.1/web/application/applicationextension.inc.php	2018-10-17 13:07:36.000000000 +0800
+++ web/application/applicationextension.inc.php	2018-12-03 17:44:30.107586019 +0800
@@ -1160,6 +1160,10 @@ class RestUtils
 				}
 				$value = DBObjectSet::FromArray($sLnkClass, $aLinks);
 			}
+			elseif($oAttDef instanceof AttributeCustomFields)
+			{
+				$value = json_decode(json_encode($value), true);
+			}
 			else
 			{
 				$value = $oAttDef->FromJSONToValue($value);
--- itop-2.5.1/web/core/restservices.class.inc.php	2018-10-17 13:07:46.000000000 +0800
+++ web/core/restservices.class.inc.php	2018-12-03 17:47:20.396272175 +0800
@@ -102,6 +102,10 @@ class ObjectResult
 					$value[] = $aLnkValues;
 				}
 			}
+			elseif($oAttDef instanceof AttributeCustomFields)
+			{
+				$value = $oObject->Get($sAttCode)->GetValues();
+			}
 			else
 			{
 				$value = $oAttDef->GetForJSON($oObject->Get($sAttCode));
