diff --git a/cmdbabstract.class.inc.php b/cmdbabstract.class.inc.php
index 4ea0dd1..5ac2943 100644
--- a/cmdbabstract.class.inc.php
+++ application/cmdbabstract.class.inc.php
@@ -1773,15 +1773,16 @@ EOF
 				break;
 				
 				case 'OQLExpression':
+
 				case 'Text':
 					$aEventsList[] ='validate';
 					$aEventsList[] ='keyup';
 					$aEventsList[] ='change';
 					$sEditValue = $oAttDef->GetEditValue($value);
+
 					$aStyles = array();
 					$sStyle = '';
 					$sWidth = $oAttDef->GetWidth('width', '');
-
 					if (!empty($sWidth))
 					{
 						$aStyles[] = 'width:'.$sWidth;
@@ -1795,6 +1796,7 @@ EOF
 					{
 						$sStyle = 'style="'.implode('; ', $aStyles).'"';
 					}
+
 					if ($oAttDef->GetEditClass() == 'OQLExpression')
 					{
 						$sTestResId = 'query_res_'.$sFieldPrefix.$sAttCode.$sNameSuffix; //$oPage->GetUniqueId();
@@ -1808,8 +1810,24 @@ EOF
 						$sAdditionalStuff = "";
 					}
 					// Ok, the text area is drawn here
-					$sHTMLValue = "<table><tr><td><textarea class=\"resizable\" title=\"$sHelpText\" name=\"attr_{$sFieldPrefix}{$sAttCode}{$sNameSuffix}\" rows=\"8\" cols=\"40\" id=\"$iId\" $sStyle>".htmlentities($sEditValue, ENT_QUOTES, 'UTF-8')."</textarea>$sAdditionalStuff</td><td>{$sValidationSpan}{$sReloadSpan}</td></tr></table>";
+					$sHTMLValue = "<div class=\"field_input_zone field_input_text\"><div class=\"f_i_text_header\"><span class=\"fullscreen_button\" title=\"".Dict::S('UI:ToggleFullScreen')."\"></span></div><textarea class=\"\" title=\"$sHelpText\" name=\"attr_{$sFieldPrefix}{$sAttCode}{$sNameSuffix}\" rows=\"8\" cols=\"40\" id=\"$iId\" $sStyle>".htmlentities($sEditValue, ENT_QUOTES, 'UTF-8')."</textarea>$sAdditionalStuff</div>{$sValidationSpan}{$sReloadSpan}";
 
+                    $oPage->add_ready_script(
+<<<EOF
+                        $('#$iId').closest('.field_input_text').find('.fullscreen_button').on('click', function(oEvent){
+                            var oOriginField = $('#$iId').closest('.field_input_text');
+                            var oClonedField = oOriginField.clone();
+                            oClonedField.addClass('fullscreen').appendTo('body');
+                            oClonedField.find('.fullscreen_button').on('click', function(oEvent){
+                                // Copying value to origin field
+                                oOriginField.find('textarea').val(oClonedField.find('textarea').val());
+                                oClonedField.remove();
+                                // Triggering change event
+                                oOriginField.find('textarea').triggerHandler('change');
+                            });
+                        });
+EOF
+                    );
 				break;
 
 				case 'CaseLog':
